container.go

```diff
@@ -452,6 +356,18 @@ func newContainer(pod *Pod, contConfig ContainerConfig) (*Container, error) {
 		c.mounts = mounts
 	}
 
+	devices, err := c.fetchDevices()
+	if err == nil {
+		c.devices = devices
+	} else {
+		// If devices were not found in storage, create Device implementations
+		// from the configuration.
```

Comment:
> If devices were not found in storage

Is that solely to handle the `create` to `start` transition ?
It would look better if we could have the same code logic for devices as we do for mounts, process, etc...i.e. we use the Container config information as the default one and it gets overwritten by what we found on storage.

Reply:
done